js_modules/dagit/packages/core/src/instance/BackfillTable.mocks.ts

```diff
@@ -336,6 +340,7 @@ export const BackfillTableFragmentCompletedAssetJobStatus: MockedResponse<Single
 export const BackfillTableFragmentCompletedOpJob: BackfillTableFragment = {
   backfillId: 'pqdiepuf',
   status: BulkActionStatus.COMPLETED,
+  isValidSerialization: true,
```

Comment:
Not a blocker, but it would be nice to add a mock for the the isValidSerialization:false case. It seems like it could be tricky for us front-end devs to see this case locally without spending a couple minutes setting up a backfill with an out-of-date def!

Reply:
sounds good, I added this test!